src/cow-react/modules/wallet/web3-react/connection/index.tsx

```diff
@@ -102,7 +105,8 @@ export function ConnectWalletOptions({ tryActivation }: { tryActivation: TryActi
     (!isInjectedMobileBrowser && <WalletConnectOption tryActivation={tryActivation} />) ?? null
 
   const fortmaticOption = (!isInjectedMobileBrowser && <FortmaticOption tryActivation={tryActivation} />) ?? null
-  const zengoOption = <ZengoOption tryActivation={tryActivation} />
+  const zengoOption = (!isInjectedMobileBrowser && <ZengoOption tryActivation={tryActivation} />) ?? null
+  const ambireOption = (!isInjectedMobileBrowser && <AmbireOption tryActivation={tryActivation} />) ?? null
```

Comment:
Since we call them like this 

```javascript
      {fortmaticOption}
      {zengoOption}
      {ambireOption}
```
I think it makes sense to have them separate for now since this might be changing

Reply:
as you wish, not a big deal